services/catalog/src/simcore_service_catalog/api/routes/services.py

```diff
@@ -76,19 +76,23 @@ def _build_cache_key(fct, *_, **kwargs):
 # (when e2e runs or by the webserver when listing projects) therefore
 # a cache is setup here
 @router.get("", response_model=list[ServiceGet], **RESPONSE_MODEL_POLICY)
-@cancellable_request
 @cached(
     ttl=LIST_SERVICES_CACHING_TTL,
     key_builder=_build_cache_key,
 )
+@cancel_on_disconnect
```

Comment:
Q: why did you move that one? just wondering if that makes sense. caching a cancellable request? should the cancellation not be on top?

Reply:
yes, my bad. good catch. I think it is safer to put it on top